src/enzo/New_Grid_WriteGrid.C

```diff
@@ -808,7 +835,34 @@ int grid::Group_WriteGrid(FILE *fptr, char *base_name, int grid_id, HDF5_hid_t f
     delete [] temp;
 
   } // end: if (NumberOfParticles > 0)
- 
+
+    /* ------------------------------------------------------------------- */
+  /* 4) Save active particle quantities. */
+
+  if (NumberOfActiveParticles > 0) {
+    /* Iterate over the enabled active particle types */
+
+    ParticleGroupID = H5Gcreate(group_id, "Active Particles", 0);
```

Comment:
Reminder - Need to make YT read the 'Active Particles' data.

Reply:
Yes :)